src/openrct2/ride/TrackDesign.cpp

```diff
@@ -1993,10 +1993,10 @@ static bool TrackDesignPlacePreview(TrackDesignState& tds, TrackDesign* td6, mon
     uint8_t backup_rotation = _currentTrackPieceDirection;
     uint32_t backup_park_flags = gParkFlags;
     gParkFlags &= ~PARK_FLAGS_FORBID_HIGH_CONSTRUCTION;
-    int32_t mapSize = gMapSize << 4;
+    auto mapSize = TileCoordsXY{ gMapSize.x / 16, gMapSize.y / 16 };
```

Comment:
This is reversed. `<< 4` corresponds with `* 16`, not `/ 16`.

Reply:
Well spotted!